### PR TITLE
[ADD]Helpdesk: add BlackList config

### PR DESCRIPTION
**Pull request by AlhMaria**
Helpdesk: create the BlackList config

image

### CODE REVIEW

**Comment by AlhMaria** [comment]
@aitorbouzas 
I think that is already solved : https://github.com/aitorbouzas/OCA-helpdesk/pull/4

File: helpdesk/models/res_config.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from odoo import fields, models, api
+
+
+class SaleConfigSettings(models.TransientModel):
```
Comment (aitorbouzas) [inline helpdesk/models/res_config.py]: HelpdeskConfigSettings

File: helpdesk/models/res_config.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    _inherit = 'res.config.settings'
+
+    # Getter / Setter Section
+    @api.model
+    def get_values(self):
+        res = super(SaleConfigSettings, self).get_values()
```
Comment (aitorbouzas) [inline helpdesk/models/res_config.py]: `super().get_values()` is enough.

File: helpdesk/models/res_config.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                number_of_minutes),
+            crm_helpdesk_improvement_number_of_tickets=int(
+                number_of_tickets
+            )
+        )
+        return res
+
+    @api.multi
+    def set_values(self):
+        super(SaleConfigSettings, self).set_values()
```
Comment (aitorbouzas) [inline helpdesk/models/res_config.py]: `super().get_values()` is enough.

File: helpdesk/__manifest__.py [hunk 4]
```diff
@@ -15,13 +15,15 @@
                 'Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/helpdesk',
     'depends': [
+        'sale',
```
Comment (aitorbouzas) [inline helpdesk/__manifest__.py]: Is this a real dependency?
This module shouldn't depend on `sale`, it must stand by itself.

**Review by aitorbouzas — CHANGES_REQUESTED** [review]
Thanks for the PR @AlhMaria !
I don't get what you are doing with this changes.
You just add some configuration to set amount of tickets permitted from the same source for creating blacklists. But that's all, isn't it?
Also, why is there a dependency with `sale`? We should avoid it in a base module.

File: helpdesk/__manifest__.py (hunk 4 above)
Comment (AlhMaria) [reply helpdesk/__manifest__.py]: You are rigth.
This module doesn't depend on sale

File: helpdesk/models/res_config.py (hunk 1 above)
Comment (AlhMaria) [reply helpdesk/models/res_config.py]: Yes

File: helpdesk/models/res_config.py (hunk 2 above)
Comment (AlhMaria) [reply helpdesk/models/res_config.py]: ok

File: helpdesk/models/res_config.py (hunk 3 above)
Comment (AlhMaria) [reply helpdesk/models/res_config.py]: ok

**Pull request closed without merging**